iModelCore/ECDb/Tests/NonPublished/InstanceReaderTest.cpp

```diff
@@ -1,2332 +1,2332 @@
-/*---------------------------------------------------------------------------------------------
```

Comment:
Does this file have any real changes or is this just line ending changes or something similar?

Reply:
Oh no, this appears to be the same thing Matt had a while back. Yeah the file has real changes, several class ids have been adjusted. You can click the little cogwheel on github and tell it to hide whitespace, then it shows the real diff.
I'll check from which line ending to which this switch was, we probably don't want to push it like this..

Reply:
I see what's happening, when I download the file from the main branch through the github website, it has newline line endings. However, when I have my local git switch to the main branch it's using CRLF, for both of the branches here.
Looking at the history, the file was changed from LF to CRLF by @mattbjordan on Aug 3 in Changeset #393 
Then it was changed back from CRLF to LF by Affan on Aug 8 in Changeset #374 (Add support for ECSQL expression tree)

Reply:
It's supposed to be CRLF. Most of the files in the repo use CRLF, and any that don't are the outliers. The reverse change keeps getting through in bigger PRs in part because Git doesn't tell you about the line-endings change locally - it only appears on Github. Everyone's VSCode line-endings settings can be different, so maybe look into adding a `.vscode/settings.json` file that helps enforce the correct line-endings? (Probably in a separate PR.) Looks like the setting is:
```json
"files.eol": "\r\n"
```